src/core_modules/capture-core/components/WorkingLists/WorkingListsBase/ListViewLoader/ListViewLoader.component.js

```diff
@@ -61,28 +59,26 @@ const useLoadView = ({
         prevTemplate: prevTemplateRef.current,
     });
     triggerLoadRef.current = triggerLoad;
-
     const cancelLoadViewIfApplicable = useCallback(() => {
         triggerLoadRef.current && onCancelLoadView && onCancelLoadView();
     }, [onCancelLoadView]);
 
     useEffect(() => {
         prevTemplateRef.current = currentTemplate;
         firstRunRef.current = false;
-
         if (triggerLoad) {
             onLoadView(currentTemplate,
                 { programId, programStageId, orgUnitId, categories },
             );
         }
         return () => cancelLoadViewIfApplicable();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Hey, 
What is the scenario where the hook behaves inconsistently due to `orgUnitId` being part of the dependency array? 
From the cases I tested, it looked like `eslint-disable-next-line` can be avoided if you check that `orgUnitId` has a value before calling the `onLoadView` function. 
Thanks!

Reply:
I think there is some advanced scenarios where we can get in trouble that might/might not be possible in the UI right now (for example: still in loading, change orgUnit comes to mind). This could probably have been refactored, but there is some complexity here that I don't want to dive into right now. I think it's fine to disable this check at times to keep the complexity down and make it easier to read.